plugins/woocommerce/src/Admin/Features/ProductBlockEditor/Init.php

```diff
@@ -370,6 +370,48 @@ public function add_product_template( $args ) {
 						),
 					),
 				),
+				array(
+					'woocommerce/product-tab',
+					array(
+						'id'    => 'organization',
+						'title' => __( 'Organization', 'woocommerce' ),
+						'order' => 15,
+					),
+					array(
+						array(
+							'woocommerce/product-section',
+							array(
+								'title' => __( 'Product catalog', 'woocommerce' ),
+							),
+							array(
+								array(
+									'woocommerce/product-category-field',
+									array(
+										'name' => 'categories',
+									),
+								),
+								array(
+									'woocommerce/product-checkbox-field',
+									array(
+										'label'    => __( 'Disable product reviews', 'woocommerce' ),
+										'property' => 'reviews_allowed',
```

Comment:
The property is in the positive... `reviews_allowed`. But, our UI is in the negative "Disable product reviews". So, we need to negate this in order for it to be saved correctly.

@jarekmorawski I know in general using negative wording in checkboxes/toggles can be confusing. Just double-checking that we want to do this.

Reply:
I see your point. The copy can be updated if you find it confusing. I changed it because it isn't very common for users to disable reviews. However, we can probably tweak the copy to `Enable reviews` (and keep the logic we currently have) and just enable it by default, letting the user opt-out if necessary. I'll update the AC.